source/Xunit.Gherkin.Quick/CoreModel/Feature/FeatureClass.cs

```diff
@@ -9,6 +9,7 @@ namespace Xunit.Gherkin.Quick
     internal sealed class FeatureClass
     {
         public string FeatureFilePath { get; }
+        public string FileNameSearchPattern { get; }
```

Comment:
Please rename to "FeatureFileSearchPattern" for clarity, otherwise it's confusing what file it is talking about.